django-transparent/docker-compose.django-transparent.yml

```diff
@@ -11,9 +11,9 @@ services:
         # We do not need a network to generate keys at all
         network_mode: "none"
         environment:
-            # INSECURE!!! You MUST define your own ACRA_MASTER_KEY
+            # INSECURE!!! You MUST define your own ACRA_CONNECTOR_MASTER_KEY
```

Comment:
wrong comment? @AlexShmelev 

`You MUST define your own ACRA_SERVER_MASTER_KEY and ACRA_CONNECTOR_MASTER_KEY`